utils/decorators/ctrlWrapper.js

```diff
@@ -1,3 +1,5 @@
+// const HttpError = require("../helpers/HttpError");
+
 const ctrlWrapeer = (ctrl) => {
```

Comment:
```suggestion
const controllerWrapper= (controller) => {
``` - краще уникати неочевидних скорочень в назвах